WebAppConfig_dockerized.json

```diff
@@ -0,0 +1,27 @@
+﻿{
```

Comment:
You may want to put it into the same directory with the other configs, and perhaps rename to something like `WebAppConfig_dockerized_local.json` to keep it distinct from the production config.